packages/data-provider/src/utils/requestValidation.ts

```diff
@@ -28,7 +28,7 @@ export function validateRequest<TBody>(
 		} catch (error) {
 			if (error instanceof ZodError) {
 				const errorMessages = error.errors
-					.map((issue: any) => `${issue.path.join('.')} is ${issue.message}`)
+					.map((issue: ZodIssue) => `${issue.path.join('.')} is ${issue.message}`)
```

Comment:
Don't need type declaration here. `error.errors` is typed as `ZodIssue[]` so we can infer the type.